zebra/zebra_vty.c

```diff
@@ -4312,7 +4312,7 @@ DEFUN(ip_table_range, ip_table_range_cmd,
 	if (!zvrf)
 		return CMD_WARNING;
 
-	if (zvrf_id(zvrf) != VRF_DEFAULT && !vrf_is_backend_netns()) {
+	if (!vrf_is_backend_netns()) {
```

Comment:
Won't it break the command for VRF-lite in the default VRF? The change makes it impossible to use this command in VRF-lite at all.
I think this change may not be needed. If the VRF is not active yet, it's vrf_id is VRF_UNKNOWN and this check will succeed.

Reply:
it will succeed, meaning that the table range config will succeed for vrfs that have vrf_unknown set when in netns mode.
correct. then, table manager works correctly from my perspective.
I close the pr.